src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -638,6 +624,20 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
     }
   }
 
+  def _matcher(value: Any) = {
+    val encoder = new JsonRowEncoder[Map[String, Any]]()
```

Comment:
Maybe make this lazy (ie, `lazy val encoder = ...`) to avoid executing it until we need it?